app/uk/gov/hmrc/gform/auth/models/RecoverAuthResult.scala

```diff
@@ -48,12 +50,24 @@ object RecoverAuthResult {
   def recoverNoActiveSession(
     request: Request[AnyContent],
     appConfig: AppConfig
-  ): PartialFunction[Throwable, AuthResult] = { case _: NoActiveSession =>
-    logger.debug("No Active Session")
-    val continueUrl = java.net.URLEncoder.encode(appConfig.`gform-frontend-base-url` + request.uri, "UTF-8")
-    val ggLoginUrl = appConfig.`government-gateway-sign-in-url`
-    val url = s"$ggLoginUrl?continue=$continueUrl"
-    AuthRedirectFlashingFormName(url)
+  ): PartialFunction[Throwable, AuthResult] = {
+    case _: NoActiveSession if request.method == "POST" =>
+      val formTemplateWithRedirect = request.attrs(FormTemplateKey)
+      val formTemplate = formTemplateWithRedirect.formTemplate
+      logger.debug("No Active Session")
+      val continueUrl = URLEncoder.encode(
+        s"${appConfig.`gform-frontend-base-url`}/submissions/new-form/${formTemplate._id.value}",
```

Comment:
Let's not hardcode `/submissions/new-form/` here. 
We can use reverse route like `routes.NewFormController.dashboard(...)` instead.